### DISCUSSION TITLE
Buderus set selFlowTemp not work

### DISCUSSION THREAD

**Discussion by Imerwin** [emsesp/EMS-ESP32#1021] [General]
Hello all,
thank you very much for all the work.

I have a Buderus Logatherm WLW196i-8 IR heat pump in use.
Unfortunately, I can not set the flow temperature.

But other values I can change vor example:
Boiler: dhw maximum temperature or
Thermostat: hc1 manual temperature 

When I change the"selected flow temperature" the new value is accepted and in the lower left corner it says green "Write command send" but the old value remains.

![image](https://user-images.githubusercontent.com/17006674/211223184-fafefdbf-2781-4b06-ae9c-a8e611622416.png)

![image](https://user-images.githubusercontent.com/17006674/211223205-2f56f458-44ce-4dad-89f0-e41aec5315d8.png)

![image](https://user-images.githubusercontent.com/17006674/211223217-c8f50424-f616-4fa3-9ec7-05dff0d223ed.png)

![image](https://user-images.githubusercontent.com/17006674/211223558-86d58838-6189-45a0-9409-e895af0433e2.png)

![image](https://user-images.githubusercontent.com/17006674/211223766-2f415591-a498-4d79-8f70-db136ed23929.png)

![image](https://user-images.githubusercontent.com/17006674/211223798-cc466ab5-29ac-494c-a746-28e177294232.png)

Device information
> EMS-ESP Version
> v3.4.4
> 
> Device (Platform / SDK)
> ESP32 / v4.4.2
> 
> System Uptime
> 004+13:06:24.853
> 
> CPU Frequency
> 240 MHz
> 
> Heap (Free / Max Alloc)
> 145.172 / 79.860 bytes
> 
> Flash Chip (Size / Speed)
> 4.194.304 bytes / 40 MHz
> 
> File System (Used / Total)
> 24.576 / 65.536 bytes (40.960 bytes free)

For my SmartHome heating control I need to be able to control the flow temperature. I hope you can help me.

Thank you and best regards

**Comment by MichaelDvP** [comment]
A thermostat is a clever electronic device. You can put in the desired room temperature and it calculates from some parameters and measurements the required flowtemp for this room temperature and sends it to the boiler. This happens in a control loop and is updated often.
